packages/shared/src/components/modals/CreateMyFeedModal.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { ModalProps } from './StyledModal';
+import { ResponsiveModal } from './ResponsiveModal';
+import styles from './AccountDetailsModal.module.css';
+import TagsFilter from '../filters/TagsFilter';
+import CreateFeedFilterButton from '../CreateFeedFilterButton';
+import PlusIcon from '../../../icons/plus.svg';
+import XIcon from '../../../icons/x.svg';
+import { Button, ButtonSize } from '../buttons/Button';
+
+export type TypeProps = {
+  type: string;
+};
+
+export type CreateMyFeedModalProps = TypeProps & ModalProps;
+
+export type LayoutModalProps = TypeProps & Partial<ModalProps>;
+
+export type CloseButtonProps = {
+  size?: ButtonSize;
+  position?: string;
+  closeButtonClassName?: string;
+} & Partial<ModalProps>;
+
+const buttonClass = {
+  v2: 'small',
+  v3: 'medium',
+  v4: 'w-40',
+  v5: 'w-40 ml-3',
+};
+
+const headerTitle = {
+  v3: 'Choose tags to follow',
+  v4: 'Feed filters',
+  v5: 'Choose tags to follow',
+};
+
+const getFooter = ({ type, onRequestClose }: LayoutModalProps) => {
+  return (
+    <footer className="flex fixed responsiveModalBreakpoint:sticky bottom-0 justify-center items-center py-3 border-t border-theme-divider-tertiary bg-theme-bg-tertiary">
+      {type === 'v4' && (
+        <CreateFeedFilterButton
+          className={classNames(buttonClass[type], 'btn-primary-cabbage')}
+        />
+      )}
+      {type === 'v5' && (
+        <>
+          <Button className="mr-3 w-40 btn-secondary" onClick={onRequestClose}>
+            Cancel
+          </Button>
+          <CreateFeedFilterButton
+            className={classNames(buttonClass[type], 'btn-primary-cabbage')}
+          />
+        </>
+      )}
```

Comment:
Can we not combine the <CreateFeedFilterButton> they seem twice the same, so we can just render the close button dynamically.

Reply:
✅

Reply:
@rebelchris yes, I would like to look at that together 🙏